File: 231111/연속되는 수 4/continuous-number4.py
N=int(input())

num=[int(input()) for _ in range(N)]

cnt,maxnum=0,0

for i in range(N):
    if i==0 or num[i]<=num[i-1]:
        cnt=1
    else:
        cnt+=1
    maxnum=max(maxnum,cnt)

print(maxnum)

File: 251207/연속되는 수 4/continuous-number4.js
const fs = require("fs");
const input = fs.readFileSync(0).toString().trim().split('\n');
const n = Number(input[0]);
const arr = input.slice(1, 1 + n).map(Number);
// Please Write your code here.

let ans=0, cnt=0;
for(let i=0; i<n; i++){
    if(i>=1 && arr[i]>arr[i-1]){
        cnt++;
    }else{
        cnt=1;
    }
    ans = Math.max(ans,cnt);
}

console.log(ans)